images/versions.sh

```diff
@@ -1,5 +1,5 @@
-export DOCKER_IMAGE_VERSION="1.3.0"
-export KAFKA_VERSION="2.4.0"
+export DOCKER_IMAGE_VERSION="1.4.0"
```

Comment:
let's update using:
https://github.com/kudobuilder/operators/tree/master/repository/kafka#upcoming-releases

as we haven't done any release beyond 1.2.0, we can still use 1.2.1 for 2.4.1 and 1.3.0 for 2.5.0